pkgs/development/libraries/portaudio/default.nix

```diff
@@ -12,7 +13,9 @@ stdenv.mkDerivation rec {
     ++ stdenv.lib.optional (!stdenv.isDarwin) alsaLib;
 
   configureFlags = stdenv.lib.optionals stdenv.isDarwin
-    [ "--build=x86_64" "--without-oss" "--enable-static" "--enable-shared" ];
+    [ "--disable-mac-universal" ];
+
+  propagatedBuildInputs = stdenv.lib.optionals stdenv.isDarwin [ AudioUnit AudioToolbox CoreAudio CoreServices Carbon ];
 
   preBuild = stdenv.lib.optionalString stdenv.isDarwin ''
     sed -i '50 i\
```

Comment:
Yep. It's needed here because Makefile is not generated until after configurePhase. We should just modify Makefile.in but the line numbers won't be the same.